src/gmail.js

```diff
@@ -280,7 +280,9 @@ var Gmail = function(localJQuery) {
     };
 
     api.check.is_thread = function() {
+        // this should match the sub_selector (nH -> if)
         var check_1 = $(".nH .if").children(":eq(1)").children().children(":eq(1)").children();
+        // Bu ->
```

Comment:
We probably don't want these comments here to be part of this PR.

Reply:
These comments were intentional as they relate back to how the selector/sub selector values are used in multiple places in the code. I had to spend some time understanding the coupling.

Reply:
Yea happy to make this more clear. Basically the `Bu` was one of the selectors and `nH` is the other. There are two "code" paths to flag this as a thread based on the selector.

Reply:
I can add something along the lines of this comment ^ if you think that will make it clear

Reply:
This looks good now.